docs/jitsi-dev.md

```diff
@@ -0,0 +1,90 @@
+# Jitsi wrapper developer docs
+
+*If you're looking for information on how to set up Jitsi in your Riot, see 
+[jitsi.md](./jitsi.md) instead.*
+
+These docs are for developers wondering how the different conference buttons work
+within Riot. If you're not a developer, you're probably looking for [jitsi.md](./jitsi.md).
```

Comment:
yes, this is duplicated. Theory is that the audience is clearly communicated using a couple tricks involving word choice, styling, and placement on the page.